package.json

```diff
@@ -159,7 +159,8 @@
 		"@wordpress/dependency-extraction-webpack-plugin": "3.2.1",
 		"@wordpress/dom": "3.27.0",
 		"@wordpress/e2e-test-utils": "10.1.0",
-		"@wordpress/e2e-tests": "4.6.0",
+		"@wordpress/e2e-test-utils-playwright": "https://github.com/woocommerce/woocommerce-blocks/files/11286971/wordpress-e2e-test-utils-playwright-0.0.0.tgz",
```

Comment:
Just a workaround. Waiting for https://github.com/WordPress/gutenberg/pull/43998

Reply:
#43998 [has been merged](https://github.com/WordPress/gutenberg/pull/43998#event-9173664175) 🎉 Thank you, @gigitux for helping draw attention to this PR 🙌🏻